test/estimation.jl

```diff
@@ -19,3 +19,54 @@ using DSP, Test
     frequencies_estimated = sort(esprit(x, M, p, Fs))
     @test isapprox(frequencies', frequencies_estimated; atol = 1e-2)
 end
+
+@testset "jacobsen" begin
+    # real input
+    fs = 100
+    t = range(0, 5, step = 1/fs)
+    fr = 28.3
+    sr = cos.(2π*fr*t .+ π/4.2)
+    f_est_real = jacobsen(sr, fs)
+    @test isapprox(f_est_real, fr, atol = 0.5)
+    @test jacobsen(zeros(10)) == 0.1
+    # complex input
+    fc = -40.3
+    sc = cis.(2π*fc*t .+ π/1.4)
+    f_est_complex = jacobsen(sc, fs)
+    @test isapprox(f_est_complex, fc, atol = 1e-2)
+    @test jacobsen(zeros(ComplexF64, 10)) == -0.5
```

Comment:
```suggestion
```
Why should the frequency of an all-zero signal depend on the type (wheher complex or not)?

Reply:
For all-zero (both real and complex signals), the DFT is constant. In this case, `argmax` returns the first array element. For real signals, the first frequency is zero (since we use `rfft`), but for complex signals it is negative Nyquist.

The purpose of these tests was to make sure all code in `jacobsen` is covered by the tests (codecov identified a branch that was not tested). This branch is taken when the DFT peak occurs in one of the two extreme frequencies. I've now added better tests for this scenario, covering both extremes of the spectrum.